lib/wait_for_socket.js

```diff
@@ -0,0 +1,30 @@
+var Promise = require('promise');
```

Comment:
looks like you converted this to using new import and didn't get rid of previous requires.